src/Kekiri/Impl/Reporting/TraceReportTarget.cs

```diff
@@ -18,7 +18,7 @@ public static IReportTarget GetInstance()
 
         public void Report(ScenarioReportingContext scenario)
         {
-            Trace.WriteLine(scenario.CreateReport());
+            Debug.WriteLine(scenario.CreateReport());
```

Comment:
Agreed. I changed it simply because Trace functionality has been broken into a separate package which would mean an extra dependency for the core Kekiri library, which I wasn't convinced was worth it. I'll rename the class.